src/ui/views/WaitForDevice.svelte

```diff
@@ -8,13 +8,15 @@
     showSelectDeviceModal
   } from "../../stores.mjs";
 
+  let initialised = false;
```

Comment:
Nitpick: We are already using the AE version `initialized`: https://github.com/ubports/ubports-installer/blob/89f4352efd7b0636dd92f73b7aaeda2419266a42/src/core/plugins/plugin.js#L48

Reply:
Ah, apologies, I should have checked. I've changed it to use `initialized` instead.